spec/support/requirements_helper.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+%i[form summary].each do |style|
+  RSpec::Matchers.define :"have_#{style}_item" do |field, key, context = {}|
+    match do |actual_issues|
+      context = context.merge(force_i18n_to_raise_missing_interpolation_args: true)
+      expected_message = I18n.t!("requirements.#{field}.#{key}.#{style}_message", context)
+      expected_message == actual_issues.items_for(field, style: style).first[:text]
```

Comment:
Are there any risks associated with the use of first here? Could it be we do a map of messages and see if expected_message is included?